daemon/mgr/container_envs.go

```diff
@@ -25,33 +24,24 @@ import (
 // /etc/profild.d/*.sh (here is pouchenv.sh). After that a process execed from outside would take env of init process,
 // then such action like reload of detailed user application could share the env.
 const (
-	// containerInstanceInfo is file in the container image.
-	// row in this file has content like `env_appDeployType = taoJavaWeb`
-	containerInstanceInfo = "/etc/instanceInfo"
 	// pouchEnvDir =
 	pouchEnvDir = "/etc/profile.d"
 	// this filepath is used in PouchContainer to store user input env via persistent file
 	pouchEnvFile = "/etc/profile.d/pouchenv.sh"
 )
 
 // updateContainerEnv update the container's envs in
-// /etc/instanceInfo and /etc/profile.d/pouchenv.sh
-// Env used by rich container.
+// /etc/profile.d/pouchenv.sh used by rich container.
 func updateContainerEnv(inputRawEnv []string, baseFs string) error {
 	var (
-		envPropertiesPath = path.Join(baseFs, containerInstanceInfo)
-		envShPath         = path.Join(baseFs, pouchEnvFile)
+		envShPath = path.Join(baseFs, pouchEnvFile)
 	)
 
 	// check the existence of related files.
 	// if dir of pouch.sh is not exist, it's unnecessary to update that files.
 	if _, err := os.Stat(path.Join(baseFs, pouchEnvDir)); err != nil {
 		return nil
 	}
-	if _, err := os.Stat(envPropertiesPath); err != nil {
-		//if etc/instanceInfo is not exist, it's unnecessary to update that file.
-		return nil
-	}
 	if _, err := os.Stat(envShPath); err != nil {
 		return fmt.Errorf("failed to stat container's env file /etc/profile.d/pouchenv.sh: %v", err)
```

Comment:
if `envShPath` is not exist, we should return nil, as you said, we just judge the file i exist.

Reply:
Using
```
	if _, err := os.Stat(envShPath); err != nil {
		logrus.Warnf("failed to stat container's env file /etc/profile.d/pouchenv.sh: %v", err)
		return nil
	}
```
? @Ace-Tang